### ISSUE TITLE
Feature: Re-fight Bosses?

### ISSUE THREAD

**Issue by RedDude** [MoArtis/RCGMod#49]
Maybe a way to reset boss encounters?
"F(5-9)" key or a interactive button in the boss lair?

**Comment by MoArtis** [comment]
Good idea! I'm guessing that force leave and re-enter the room should do the trick.
